### PR TITLE
Extend support for more websites

### PR DESCRIPTION
**Pull request by charsleysa**
Related #84

### CODE REVIEW

File: packages/metascraper-image/index.js [hunk 1]
```diff
@@ -31,6 +31,8 @@ module.exports = () => ({
     wrap($ => $('meta[name="sailthru.image"]').attr('content')),
     wrap($ => getValue($, $('article img[src]'), getSrc)),
     wrap($ => getValue($, $('#content img[src]'), getSrc)),
+    wrap($ => getValue($, $('.post-image img[src]'), getSrc)),
+    wrap($ => getValue($, $('.img-responsive[src]'), getSrc)),
```
[the remark this answers is not in the document]
Comment (charsleysa) [reply packages/metascraper-image/index.js]: Don't we want a specific selector so we get the main image and not a smaller unrelated image?

[the remark this answers is not in the document]
Comment (Kikobeats) [reply packages/metascraper-image/index.js]: Need to validate it's useful the major website provided, not just one

[the remark this answers is not in the document]
Comment (charsleysa) [reply packages/metascraper-image/index.js]: What's the process for that?

[the remark this answers is not in the document]
Comment (Kikobeats) [reply packages/metascraper-image/index.js]: adding more popular on tests and check the value extract for the prop is or not correct before or after add the rule.

BTW, I think this is a good scenario for test Microlink API custom rules support

[the remark this answers is not in the document]
Comment (Kikobeats) [reply packages/metascraper-image/index.js]: Probably this month 🙂

**Comment by charsleysa** [comment]
Closing in favor of Microlink API custom rules.

**Pull request closed without merging**